src/adaptation/AccessCheck.cc

```diff
@@ -87,11 +87,11 @@ Adaptation::AccessCheck::usedDynamicRules()
 
     DynamicGroupCfg services;
     if (!ah->extractFutureServices(services)) { // clears history
-        debugs(85,9, HERE << "no service-proposed rules stored");
+        debugs(85,9, "no service-proposed rules stored");
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.